src/gui/qgsmaptooldigitizefeature.h

```diff
@@ -91,7 +91,7 @@ class GUI_EXPORT QgsMapToolDigitizeFeature : public QgsMapToolCaptureLayerGeomet
      * Called when the feature has been digitized.
      * \param geometry the digitized geometry
      */
-    void layerGeometryCaptured( const QgsGeometry &geometry ) override FINAL;
+    void layerGeometryCaptured( const QgsGeometry &geometry ) override final;
```

Comment:
Wouldn't it be enough to just remove the redundant override keyword?

```suggestion
    void layerGeometryCaptured( const QgsGeometry &geometry ) FINAL;
```